demisto_sdk/commands/content_graph/parsers/pack.py

```diff
@@ -388,12 +396,33 @@ def field_mapping(self):
             "default_data_source_id": "defaultDataSource",
         }
 
-    # TODO - implementing all those 3 methods
-    def validate_structure(self) -> List[StructureError]:
-        return []
-
     def raw_data(self) -> dict:
         raise NotImplementedError
 
+    @property
     def strict_object(self):
-        raise NotImplementedError
+        raise NotImplementedError("This object has a different behavior")
+
+    def validate_structure(self) -> List[StructureError]:
+        """
+        This method uses the parsed data and attempts to build a Pydantic (strict) object from it.
+        Whenever the data and schema mismatch, we store the error using the 'structure_errors' attribute,
+        which will be read during the ST110 validation run.
+        In Pack, we need to check two files: the metadata and the RNs json files, so we override the
+        method for combing all the pydantic errors from the both files.
+        """
+        directory_pydantic_error = []
+        for pack_files_or_directories in self.path.iterdir():
```

Comment:
instead of iterating over everything, always check the pack metadata, and do `for file in self.path.rglob("ReleaseNotes/*.json")` to check all the jsons under that folder, if there are any.